test/cases/grid-template-areas.out.css

```diff
@@ -32,32 +32,20 @@
     grid-area: foot;
 }
 
-.f {
-    -ms-grid-row: 3;
-    -ms-grid-column: 2;
-    grid-area: foot;
-}
-
 @media (max-width: 1000px) {
     .a {
         grid-template-areas: "head main";
     }
-
     .b {
-        -ms-grid-row: 1;
-        -ms-grid-column: 1;
-        grid-area: head;
+        -ms-grid-row:
+        1;
```

Comment:
There is a little problem here with whitespaces. Is it needed to merge raws from different rules/decls for the better result?

Reply:
Just trim spaces around value and remove `raws.between`

Reply:
Used `raws.between` of the origin declaration